### PR TITLE
Curating b4x as a new topic

### PR DESCRIPTION
**Pull request by pyhoon**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [✔ ] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [✔ ] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [✔ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [✔ ] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [✔ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [✔ ] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> B4X is a programming language and tool to develop cross platform apps. There are more and more repositories created on this topic.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of your proposed changes.

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Review by ahmadSS85 — APPROVED** [review]

**Review by ahmadSS85 — APPROVED** [review]

**Review by pyhoon — COMMENTED** [review]
Fixed typo for logo

File: topics/b4x/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+display_name: B4X
```
Comment (ahmadSS85) [inline topics/b4x/index.md:2]: Ahmad

**Review by ahmadSS85 — APPROVED** [review]

File: topics/b4x/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (pyhoon) [reply topics/b4x/index.md:2]: Thanks

Comment (pyhoon) [reply topics/b4x/index.md:2]: Thanks

**Review by pyhoon — COMMENTED** [review]
Update index.md

**Review by pyhoon — COMMENTED** [review]
Ok

**Pull request closed without merging**